app/models/xml_source_processor.rb

```diff
@@ -72,37 +72,35 @@ def validate_links(text)
         end
       end
     end
-#    return errors.size > 0
+    #    return errors.size > 0
   end
 
   ##############################################
   # All code to convert transcriptions from source
   # format to canonical xml format belongs here.
-  #
-  #
   ##############################################
   def process_source
     if @text_dirty
       self.xml_text = wiki_to_xml(self, Page::TEXT_TYPE::TRANSCRIPTION)
     end
 
     if @translation_dirty
-      self.xml_translation = wiki_to_xml(self, Page::TEXT_TYPE::TRANSLATION)      
+      self.xml_translation = wiki_to_xml(self, Page::TEXT_TYPE::TRANSLATION)
     end
   end
 
   def wiki_to_xml(page, text_type)
 
     subjects_disabled = page.collection.subjects_disabled
-    
+
     source_text = case text_type
-    when Page::TEXT_TYPE::TRANSCRIPTION
-      page.source_text
-    when Page::TEXT_TYPE::TRANSLATION
-      page.source_translation
-    else
-      ""
-    end
+                  when Page::TEXT_TYPE::TRANSCRIPTION
```

Comment:
These are either hard tabs or eight-space indentation and need to be replaced by two-space indentation.

Reply:
@saracarl Should we? My editor (Vim) disagrees and says the line is indented properly.